frontend/packages/kubevirt-plugin/src/components/modals/add-template-source/add-template-source.tsx

```diff
@@ -115,13 +122,15 @@ export const AddTemplateSourceModal: React.FC<ModalComponentProps &
       }
       close();
     } catch ({ message }) {
-      setError(message || 'Could not create persistent volume claim.');
+      setError(message || t('kubevirt-plugin~Could not create persistent volume claim.'));
```

Comment:
this wont trigger translation when user changes language.

See my PR where I created a hook to handle such cases https://github.com/openshift/console/pull/7349/files#diff-b9e2d080ccc211c6940ba931b9cc4ced54956b21995895592f1517172738a2a7R80